Makefile

```diff
@@ -115,16 +148,21 @@ docker-gpu: docker-build-gpu docker-push-gpu
 docker-gpu-dev: docker-build-gpu docker-push-gpu-dev
 
 docker-tag:
-	docker tag pachyderm/pachd pachyderm/pachd:$(VERSION)
-	docker tag pachyderm/worker pachyderm/worker:$(VERSION)
-	docker tag pachyderm/pachctl pachyderm/pachctl:$(VERSION)
-	docker tag pachyderm/pachtf pachyderm/pachtf:$(VERSION)
+	docker tag pachyderm/pachd:latest pachyderm/pachd:$(VERSION)-$(TARGET_ARCH)
+	docker tag pachyderm/worker:latest pachyderm/worker:$(VERSION)-$(TARGET_ARCH)
+	docker tag pachyderm/pachctl:latest pachyderm/pachctl:$(VERSION)-$(TARGET_ARCH)
+
 
 docker-push: docker-tag
-	$(SKIP) docker push pachyderm/pachd:$(VERSION)
-	$(SKIP) docker push pachyderm/worker:$(VERSION)
-	$(SKIP) docker push pachyderm/pachctl:$(VERSION)
-	$(SKIP) docker push pachyderm/pachtf:$(VERSION)
+	$(SKIP) docker push pachyderm/pachd:$(VERSION)-$(TARGET_ARCH)
+	$(SKIP) docker push pachyderm/worker:$(VERSION)-$(TARGET_ARCH)
+	$(SKIP) docker push pachyderm/pachctl:$(VERSION)-$(TARGET_ARCH)
+	#$(SKIP) manifest create pachyderm/pachd:$(VERSION) docker.io/pachyderm/pachd:$(VERSION)-amd64 docker.io/pachyderm/pachd:$(VERSION)-arm64
```

Comment:
pachtf?

Remove all the manifest commands

Reply:
i'll add the pachtf lines..why are we removing the manifest lines?

Reply:
These seem like permanently skipped. If so, remove the dead code.